### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by FreeKopcap** [Chia-Network/chia-blockchain#14608]
### What happened?

Can not see some NFTs in GUI.
Can not choose proper NFT wallet in GUI.
When All NFTs chosen - can see only NFTs from the last NFT wallet (ID=9).
In CLI can list all NFTs from both wallets ID=7 and ID=9 (see output in *Relevant log output* section).
 
![NFTs GUI](https://user-images.githubusercontent.com/125664809/219658385-da93ec4c-813e-4167-8b00-b24cd588d4b4.png)

### Version

1.7.0

### What platform are you using?

macOS

### What ui mode are you using?

GUI

### Relevant log output

```shell
*Wallets:*
~ chia wallet show

Active Wallet Key (*):
   -Fingerprint:           486241739
   -Label:                 NFT
   -Sync Status:           Synced

Wallet Keys:
(...skipped...)
4)  * 486241739    NFT
(...skipped...)
Choose a wallet key [1-5] ('q' to quit, or Enter to use 486241739):
Wallet height: 3262143
Sync status: Synced
Balances, fingerprint: 486241739

Chia Wallet:
   -Total Balance:         1.537574610995 xch (1537574610995 mojo)
   -Pending Total Balance: 1.537574610995 xch (1537574610995 mojo)
   -Spendable:             1.537574610995 xch (1537574610995 mojo)
   -Type:                  STANDARD_WALLET
   -Wallet ID:             1

DID did:chia:14gvtyxs0l40s6cqqvjxw9cwx0sk2lgds655hehezspaezrelk2zqvtshxp:
   -Total Balance:         1.0
   -Pending Total Balance: 1.0
   -Spendable:             1.0
   -Type:                  DECENTRALIZED_ID
   -DID ID:                did:chia:14gvtyxs0l40s6cqqvjxw9cwx0sk2lgds655hehezspaezrelk2zqvtshxp
   -Wallet ID:             4

Spacebucks:
   -Total Balance:         22.906  (22906 mojo)
   -Pending Total Balance: 22.906  (22906 mojo)
   -Spendable:             22.906  (22906 mojo)
   -Type:                  CAT
   -Asset ID:              a628c1c2c6fcb74d53746157e438e108eab5c0bb3e5c80ff9b1910b3e4832913
   -Wallet ID:             6

NFT Wallet:
   -Total Balance:         0.0
   -Pending Total Balance: 0.0
   -Spendable:             0.0
   -Type:                  NFT
   -Wallet ID:             7

NFT Wallet:
   -Total Balance:         0.0
   -Pending Total Balance: 0.0
   -Spendable:             0.0
   -Type:                  NFT
   -DID ID:                did:chia:14gvtyxs0l40s6cqqvjxw9cwx0sk2lgds655hehezspaezrelk2zqvtshxp
   -Wallet ID:             9
====
*Wallet ID 7 (invisible):*
~ chia wallet nft list -f 486241739 -i 7

NFT identifier:            nft1fq4dd0xnn7r82c4qlsuh04uy32nlkkmymvawnpwsj5k37mmlay8q304af6
Launcher coin ID:          482ad6bcd39f867562a0fc3977d7848aa7fb5b64db3ae985d0952d1f6f7fe90e
Launcher puzhash:          eff07522495060c066f66f32acc2a77e3a3e737aca8baea4d1a64ea4cdc13da9
Current NFT coin ID:       fb9fe0e1e4e331cc0a3f6755f4e0eb97725eebb3b4834f6c91dff928884bc39e
On-chain data/info:        ((117 "https://i4d73ggyrcgzkqz4az54i3k5ax6exm5jpxcwxxefoqj5dkdymh3a.arweave.net/Rwf9mNiIjZVDPAZ7xG1dBfxLs6l9xWvchXQT0ah4YfY") (104 . 0xc4445f01eef3924ac193de1512bf735b020b974145e796a85a5d28a6a44a20bd) (28021 "https://3qsceeqbbmhuwkovabcu5ubsic4ouxjqccnksbg335fwl7qyd4.arweave.net/3CQiEgELD0sp1QBF_TtAyQLjqXTAQmqkE299LZf4YH0") (28008 . 0xfdc621058d8c2ed870e444ea0db401727052ab2e6c80feb1b0c74e1b09f968d5) (27765) (27752 . 0x00) (29550 . 1) (29556 . 1))
Owner DID:                 None
Minter DID:                did:chia:1pra82ylsf8axnyzz45evdnktg0f4pc7el4xjmsdfn4mwfn5t945qgs072h
Royalty percentage:        500
Royalty puzhash:           08bac640302211ed86ef0af1112668379419f7e8dd034464b2c196bd963b531b
NFT content hash:          c4445f01eef3924ac193de1512bf735b020b974145e796a85a5d28a6a44a20bd
Metadata hash:             fdc621058d8c2ed870e444ea0db401727052ab2e6c80feb1b0c74e1b09f968d5
License hash:              00
NFT edition total:         1
Current NFT number in the edition: 1
Metadata updater puzhash:  fe8a4b4e27a2e29a4d3fc7ce9d527adbcaccbab6ada3903ccf3ba9a769d2d78b
NFT minting block height:  2238755
Inner puzzle supports DID: True
NFT is pending for a transaction: False

URIs:
   https://i4d73ggyrcgzkqz4az54i3k5ax6exm5jpxcwxxefoqj5dkdymh3a.arweave.net/Rwf9mNiIjZVDPAZ7xG1dBfxLs6l9xWvchXQT0ah4YfY

Metadata URIs:
   https://3qsceeqbbmhuwkovabcu5ubsic4ouxjqccnksbg335fwl7qyd4.arweave.net/3CQiEgELD0sp1QBF_TtAyQLjqXTAQmqkE299LZf4YH0

License URIs:

NFT identifier:            nft14hmyhsqrpswv98feqdzajetafheprncv3tj0u5tmvwqy5kmtln4qyj45ma
Launcher coin ID:          adf64bc0030c1cc29d390345d9657d4df211cf0c8ae4fe517b63804a5b6bfcea
Launcher puzhash:          eff07522495060c066f66f32acc2a77e3a3e737aca8baea4d1a64ea4cdc13da9
Current NFT coin ID:       a82f328bf1f072f3b5e6c3895db15b6152f7168a4f182919bc5f8f67d784974d
On-chain data/info:        ((117 "https://qrau5rec6sr5ykanwp4kfawzomklgzq3ao7aw32h2dqjfmemql6a.arweave.net/hEFOxIL0o9woDbP4ooLZcxSzZhsDvgtvR9DgkrCMgvw") (104 . 0xcc2822680b70ffb302636242ff68c4407b83bafa5b7eb6a85e23b6d3fc5c4183) (28021 "https://amwzq2qyjafkcj25k6urcr4kfliownbxpqkawnwg4hbdzfczdb7q.arweave.net/Ay2YahhICqEnXVepEUeKKtDrNDd8FAs2xuHCPJRZGH8") (28008 . 0x30f467f3da4a3bce6393b6a3d9f1f3785a79af5c0c1d2520f0273f27ce233540) (27765) (27752 . 0x00) (29550 . 1) (29556 . 1))
Owner DID:                 None
Minter DID:                did:chia:1pra82ylsf8axnyzz45evdnktg0f4pc7el4xjmsdfn4mwfn5t945qgs072h
Royalty percentage:        500
Royalty puzhash:           08bac640302211ed86ef0af1112668379419f7e8dd034464b2c196bd963b531b
NFT content hash:          cc2822680b70ffb302636242ff68c4407b83bafa5b7eb6a85e23b6d3fc5c4183
Metadata hash:             30f467f3da4a3bce6393b6a3d9f1f3785a79af5c0c1d2520f0273f27ce233540
License hash:              00
NFT edition total:         1
Current NFT number in the edition: 1
Metadata updater puzhash:  fe8a4b4e27a2e29a4d3fc7ce9d527adbcaccbab6ada3903ccf3ba9a769d2d78b
NFT minting block height:  2270243
Inner puzzle supports DID: True
NFT is pending for a transaction: False

URIs:
   https://qrau5rec6sr5ykanwp4kfawzomklgzq3ao7aw32h2dqjfmemql6a.arweave.net/hEFOxIL0o9woDbP4ooLZcxSzZhsDvgtvR9DgkrCMgvw

Metadata URIs:
   https://amwzq2qyjafkcj25k6urcr4kfliownbxpqkawnwg4hbdzfczdb7q.arweave.net/Ay2YahhICqEnXVepEUeKKtDrNDd8FAs2xuHCPJRZGH8

License URIs:

NFT identifier:            nft1gg77f57gpm9xajy9fftl86xnj88eyqf2mfe79hg64aqdjurygrcqr56t7j
Launcher coin ID:          423de4d3c80eca6ec8854a57f3e8d391cf92012ada73e2dd1aaf40d9706440f0
Launcher puzhash:          eff07522495060c066f66f32acc2a77e3a3e737aca8baea4d1a64ea4cdc13da9
Current NFT coin ID:       1486e906393c5275f0aae593f9473db18cd26e21334da83d2c596bf26690d7f0
On-chain data/info:        ((117 "https://q2fomvukmjh4ovamviysrqe5p65uqc25mq2es3k3a6mjvmdm.arweave.net/hormVopiT8dUDKoxKMCdf7t_IC11kN_EltWweYmrBsA") (104 . 0xf3d61557924d4afe8d5f1855ea066e3115a45311f3b1664d70c6715d756208c9) (28021 "https://5e67fgbl3ivvj7cjdzrjiasoones2eeulljrsxtjk5znltux.arweave.net/6T3ymCvaK1T8SR5ilAJOc0ktEJR-a_0xleaVdy1c6XM") (28008 . 0x6d5cb9ed9d04cf1b00e002ca8f3d9e8c86c46e2f61ab4dacb3080f33a2d5d6ca) (27765) (27752 . 0x00) (29550 . 1) (29556 . 1))
Owner DID:                 None
Minter DID:                did:chia:1pra82ylsf8axnyzz45evdnktg0f4pc7el4xjmsdfn4mwfn5t945qgs072h
Royalty percentage:        500
Royalty puzhash:           08bac640302211ed86ef0af1112668379419f7e8dd034464b2c196bd963b531b
NFT content hash:          f3d61557924d4afe8d5f1855ea066e3115a45311f3b1664d70c6715d756208c9
Metadata hash:             6d5cb9ed9d04cf1b00e002ca8f3d9e8c86c46e2f61ab4dacb3080f33a2d5d6ca
License hash:              00
NFT edition total:         1
Current NFT number in the edition: 1
Metadata updater puzhash:  fe8a4b4e27a2e29a4d3fc7ce9d527adbcaccbab6ada3903ccf3ba9a769d2d78b
NFT minting block height:  2269683
Inner puzzle supports DID: True
NFT is pending for a transaction: False

URIs:
   https://q2fomvukmjh4ovamviysrqe5p65uqc25mq2es3k3a6mjvmdm.arweave.net/hormVopiT8dUDKoxKMCdf7t_IC11kN_EltWweYmrBsA

Metadata URIs:
   https://5e67fgbl3ivvj7cjdzrjiasoones2eeulljrsxtjk5znltux.arweave.net/6T3ymCvaK1T8SR5ilAJOc0ktEJR-a_0xleaVdy1c6XM

License URIs:

NFT identifier:            nft1zmvat73s9grlzrswmxxj0yl2hkzc4sans0un52f7dkuv6ekuexwqgcwxnx
Launcher coin ID:          16d9d5fa302a07f10e0ed98d2793eabd858ac3b383f93a293e6db8cd66dcc99c
Launcher puzhash:          eff07522495060c066f66f32acc2a77e3a3e737aca8baea4d1a64ea4cdc13da9
Current NFT coin ID:       90b42a5a23759ed9bf99ad2f6739133c7a4b72f6f8ce44f23f240346c8019207
On-chain data/info:        ((117 "https://qpmzby3caz4ypnd66dveouvguqgi7p4cwe6olkbw6amc666o.arweave.net/g9mQ42IGeYe0-_fvDqR1KmpAyPv4KxPOWoNvAYL3vOw") (104 . 0x49f85909f12335afcd18132fd61d2a20bd991fd94deb1c11cc0a75a2388e9d7e) (28021 "https://nuqzfi5b4265uwugkhfaswg2xavy3hz3g7huph5iqqddopte.arweave.net/bS-GSo6HmvdpahlHKCVjauCuNnzs3-z0efqIQGNz5kQ") (28008 . 0x94ad2bc06cedf620e1c7553c2539a672ae967cf44a82b83022e94a4230b429b8) (27765) (27752 . 0x00) (29550 . 1) (29556 . 1))
Owner DID:                 None
Minter DID:                did:chia:1pra82ylsf8axnyzz45evdnktg0f4pc7el4xjmsdfn4mwfn5t945qgs072h
Royalty percentage:        500
Royalty puzhash:           08bac640302211ed86ef0af1112668379419f7e8dd034464b2c196bd963b531b
NFT content hash:          49f85909f12335afcd18132fd61d2a20bd991fd94deb1c11cc0a75a2388e9d7e
Metadata hash:             94ad2bc06cedf620e1c7553c2539a672ae967cf44a82b83022e94a4230b429b8
License hash:              00
NFT edition total:         1
Current NFT number in the edition: 1
Metadata updater puzhash:  fe8a4b4e27a2e29a4d3fc7ce9d527adbcaccbab6ada3903ccf3ba9a769d2d78b
NFT minting block height:  2288870
Inner puzzle supports DID: True
NFT is pending for a transaction: False

URIs:
   https://qpmzby3caz4ypnd66dveouvguqgi7p4cwe6olkbw6amc666o.arweave.net/g9mQ42IGeYe0-_fvDqR1KmpAyPv4KxPOWoNvAYL3vOw

Metadata URIs:
   https://nuqzfi5b4265uwugkhfaswg2xavy3hz3g7huph5iqqddopte.arweave.net/bS-GSo6HmvdpahlHKCVjauCuNnzs3-z0efqIQGNz5kQ

License URIs:

NFT identifier:            nft1ryft7n0rj8ag64kfru7uxh5g7tnaf568z5patc50rz4dwyn59rwsgdqj8j
Launcher coin ID:          1912bf4de391fa8d56c91f3dc35e88f2e7d4d3471503d5e28f18aad7127428dd
Launcher puzhash:          eff07522495060c066f66f32acc2a77e3a3e737aca8baea4d1a64ea4cdc13da9
Current NFT coin ID:       2d7a1b0296f86d5995913a5098d06c310146732ab86c985d29140a33b8f9a9d9
On-chain data/info:        ((117 "https://vttpfuzx2a2qgljdu4qurc2huumwxkgfjpsjafizxcubtcgf3e.arweave.net/rOby0zfQNQMtI6chSItHpRlrqMVL5JAV-GbioGYjF2Y") (104 . 0x54a30eba534bf33f30d041b2de4ef2c5d3d14e2d3ad9bf8b24b489a3ae97ec8c) (28021 "https://sygsr5sy5u4qyty6vyit3agnoegn4lucstmbcw6o5vzlgihe.arweave.net/lg0_o9ljtOQxPHq4RPYD-NcQzeLoKU2BFbzu1ysyDkc") (28008 . 0x7da994d9825d6114bd66d4865492138bf557dcaa18b4002ee0bcb90b25d31a82) (27765) (27752 . 0x00) (29550 . 1) (29556 . 1))
Owner DID:                 None
Minter DID:                did:chia:1pra82ylsf8axnyzz45evdnktg0f4pc7el4xjmsdfn4mwfn5t945qgs072h
Royalty percentage:        500
Royalty puzhash:           08bac640302211ed86ef0af1112668379419f7e8dd034464b2c196bd963b531b
NFT content hash:          54a30eba534bf33f30d041b2de4ef2c5d3d14e2d3ad9bf8b24b489a3ae97ec8c
Metadata hash:             7da994d9825d6114bd66d4865492138bf557dcaa18b4002ee0bcb90b25d31a82
License hash:              00
NFT edition total:         1
Current NFT number in the edition: 1
Metadata updater puzhash:  fe8a4b4e27a2e29a4d3fc7ce9d527adbcaccbab6ada3903ccf3ba9a769d2d78b
NFT minting block height:  2270316
Inner puzzle supports DID: True
NFT is pending for a transaction: False

URIs:
   https://vttpfuzx2a2qgljdu4qurc2huumwxkgfjpsjafizxcubtcgf3e.arweave.net/rOby0zfQNQMtI6chSItHpRlrqMVL5JAV-GbioGYjF2Y

Metadata URIs:
   https://sygsr5sy5u4qyty6vyit3agnoegn4lucstmbcw6o5vzlgihe.arweave.net/lg0_o9ljtOQxPHq4RPYD-NcQzeLoKU2BFbzu1ysyDkc

License URIs:

NFT identifier:            nft1y23jqxwkcfzga3tyq3fhcdqeegzpswvqz2qd0ucm9kwuvmljws9slkr9fj
Launcher coin ID:          22a32019d6c2448ec56404537c3419ca041839801280d7f31b2d9dc66ff2740b
Launcher puzhash:          eff07522495060c066f66f32acc2a77e3a3e737aca8baea4d1a64ea4cdc13da9
Current NFT coin ID:       be0fcd6f5d177e555e46a8932b06dfbb099d197c4e7b958e56e4d24aa100343b
On-chain data/info:        ((117 "https://fewka7w5rgx3pnhttojdiobumctq6yljhoocyienlcc7mtdl.arweave.net/KS-ygft2Jr7e085uSNDg0YKcPY_Wk7nCwgjViF9kxr8") (104 . 0x9c7e5aaba2ac1d8245300c62bd49553a95c04f7e441d01ed5b5c75c0aadb4f5c) (28021 "https://abrsbmmxv6zrwo3norps6z7wbknyfwwimo7ufjlvpbbirpg4.arweave.net/AG_MgsZevsxs7bXRfL2f2Cpu-C2shjv0KldXhCiLzcU") (28008 . 0xe23a6620a2e850b1d41514d66edca9b252dfed50811e206f8e37b791fd33dc69) (27765) (27752 . 0x00) (29550 . 1) (29556 . 1))
Owner DID:                 None
Minter DID:                did:chia:1pra82ylsf8axnyzz45evdnktg0f4pc7el4xjmsdfn4mwfn5t945qgs072h
Royalty percentage:        500
Royalty puzhash:           08bac640302211ed86ef0af1112668379419f7e8dd034464b2c196bd963b531b
NFT content hash:          9c7e5aaba2ac1d8245300c62bd49553a95c04f7e441d01ed5b5c75c0aadb4f5c
Metadata hash:             e23a6620a2e850b1d41514d66edca9b252dfed50811e206f8e37b791fd33dc69
License hash:              00
NFT edition total:         1
Current NFT number in the edition: 1
Metadata updater puzhash:  fe8a4b4e27a2e29a4d3fc7ce9d527adbcaccbab6ada3903ccf3ba9a769d2d78b
NFT minting block height:  2270274
Inner puzzle supports DID: True
NFT is pending for a transaction: False

URIs:
   https://fewka7w5rgx3pnhttojdiobumctq6yljhoocyienlcc7mtdl.arweave.net/KS-ygft2Jr7e085uSNDg0YKcPY_Wk7nCwgjViF9kxr8

Metadata URIs:
   https://abrsbmmxv6zrwo3norps6z7wbknyfwwimo7ufjlvpbbirpg4.arweave.net/AG_MgsZevsxs7bXRfL2f2Cpu-C2shjv0KldXhCiLzcU

License URIs:

NFT identifier:            nft1eta92dzzrzggyyrvy0wrrsvxjqh7t3t3z0tfxtkltrn7clum3jhqv9jplg
Launcher coin ID:          cafa553442189082106c23dc31c186902fe5c57113d6932edf58e7ec7f9b8cae
Launcher puzhash:          eff07522495060c066f66f32acc2a77e3a3e737aca8baea4d1a64ea4cdc13da9
Current NFT coin ID:       2b81cd420e282b5a19c0f21219cf3dd92da5c4ea4cf7ca8308a378b32f85985f
On-chain data/info:        ((117 "https://63u7evpgz4hh6fevtmwoolfqhfgfq5vynj7kkrawpozbep3wje.arweave.net/9unyVebPDn8UlZss5yywOUx-YdrhqfqVEFnuyEj92Sc") (104 . 0xfe5286b17e4a14f06d805570d5a0c961662e72ca6b1b9a8a34addd95497db721) (28021 "https://cy3i4jnlw4bmymzkukz232tfnlcrygcmaktn4ltohk4qjidq.arweave.net/FjaOJau3AswzKq_Kzre_plasUcGEwCpt4u-bjq5BKBw") (28008 . 0x7011af49042dd60703839ec49b07f00e09eff1610b006032028048920eb04e37) (27765) (27752 . 0x00) (29550 . 1) (29556 . 1))
Owner DID:                 None
Minter DID:                did:chia:1pra82ylsf8axnyzz45evdnktg0f4pc7el4xjmsdfn4mwfn5t945qgs072h
Royalty percentage:        500
Royalty puzhash:           08bac640302211ed86ef0af1112668379419f7e8dd034464b2c196bd963b531b
NFT content hash:          fe5286b17e4a14f06d805570d5a0c961662e72ca6b1b9a8a34addd95497db721
Metadata hash:             7011af49042dd60703839ec49b07f00e09eff1610b006032028048920eb04e37
License hash:              00
NFT edition total:         1
Current NFT number in the edition: 1
Metadata updater puzhash:  fe8a4b4e27a2e29a4d3fc7ce9d527adbcaccbab6ada3903ccf3ba9a769d2d78b
NFT minting block height:  2269692
Inner puzzle supports DID: True
NFT is pending for a transaction: False

URIs:
   https://63u7evpgz4hh6fevtmwoolfqhfgfq5vynj7kkrawpozbep3wje.arweave.net/9unyVebPDn8UlZss5yywOUx-YdrhqfqVEFnuyEj92Sc

Metadata URIs:
   https://cy3i4jnlw4bmymzkukz232tfnlcrygcmaktn4ltohk4qjidq.arweave.net/FjaOJau3AswzKq_Kzre_plasUcGEwCpt4u-bjq5BKBw

License URIs:
=====
*Wallet ID=9 (visible):*
➜  ~ chia wallet nft list -f 486241739 -i 9

NFT identifier:            nft1qaxvgkqjr7pdt6cvxykfznmgflh4d8pze9tn3e4f4tzr8amkv9jsemps8x
Launcher coin ID:          074cc458121f82d5eb0c312c914f684fef569c22c95738e6a9aac433f7766165
Launcher puzhash:          eff07522495060c066f66f32acc2a77e3a3e737aca8baea4d1a64ea4cdc13da9
Current NFT coin ID:       07f58fe34c73c7cf95b9c058f0f9718614720e0a6c3e3ee4832838b45682661c
On-chain data/info:        ((117 "https://i.ibb.co/rx9wncS/Kopcap.png" "https://i.postimg.cc/8567VyG7/Kopcap.png") (104 . 0x1ce8a192c0f392dc982a8850b5a3078ee2de825da528b5832c3d42a9345adc7d) (28021) (27765 "https://raw.githubusercontent.com/Chia-Network/chia-blockchain/main/LICENSE") (29550 . 1) (29556 . 1) (27752 . 0x30a358857da6b49f57cfe819c1ca43bfe007f528eb784df5da5cb64577e0ffc6))
Owner DID:                 did:chia:14gvtyxs0l40s6cqqvjxw9cwx0sk2lgds655hehezspaezrelk2zqvtshxp
Minter DID:                did:chia:1rsvrxwmk3ucyjzyme6vxuztjdqgqqrr7v09mjahq09cddur5a5rqdj66v3
Royalty percentage:        500
Royalty puzhash:           63f9abf4218b9af6a88e61a3b1e533142ce5b4bc2347e1732fbc181d6bd17cb8
NFT content hash:          1ce8a192c0f392dc982a8850b5a3078ee2de825da528b5832c3d42a9345adc7d
Metadata hash:
License hash:              30a358857da6b49f57cfe819c1ca43bfe007f528eb784df5da5cb64577e0ffc6
NFT edition total:         1
Current NFT number in the edition: 1
Metadata updater puzhash:  fe8a4b4e27a2e29a4d3fc7ce9d527adbcaccbab6ada3903ccf3ba9a769d2d78b
NFT minting block height:  2276153
Inner puzzle supports DID: True
NFT is pending for a transaction: False

URIs:
   https://i.ibb.co/rx9wncS/Kopcap.png
   https://i.postimg.cc/8567VyG7/Kopcap.png

Metadata URIs:

License URIs:
   https://raw.githubusercontent.com/Chia-Network/chia-blockchain/main/LICENSE

NFT identifier:            nft1lw2fz0f7td78zehtu5ckm2kw9040svzjmcqlmhkssx9l5708xwysrwlhyx
Launcher coin ID:          fb94913d3e5b7c7166ebe5316daace2beaf83052de01fdded0818bfa79e73389
Launcher puzhash:          eff07522495060c066f66f32acc2a77e3a3e737aca8baea4d1a64ea4cdc13da9
Current NFT coin ID:       c4f0fd03ed116e5071d6e1a25d9f1d9238097f2bf1011b260820edebf85d8cac
On-chain data/info:        ((117 "https://i.ibb.co/BtKb8sh/Block-Chian.png") (104 . 0x1dfcc4292d6424b48864050198acf7f14486a006109dc15e8bfca4cf349cc92a) (28021) (27765) (29550 . 1) (29556 . 1))
Owner DID:                 did:chia:14gvtyxs0l40s6cqqvjxw9cwx0sk2lgds655hehezspaezrelk2zqvtshxp
Minter DID:                None
Royalty percentage:        0
Royalty puzhash:           51dd6698ff88bcd21e6b9817897d34437ff3717aca3ad2c7856874f31b5ebfd0
NFT content hash:          1dfcc4292d6424b48864050198acf7f14486a006109dc15e8bfca4cf349cc92a
Metadata hash:
License hash:
NFT edition total:         1
Current NFT number in the edition: 1
Metadata updater puzhash:  fe8a4b4e27a2e29a4d3fc7ce9d527adbcaccbab6ada3903ccf3ba9a769d2d78b
NFT minting block height:  2274817
Inner puzzle supports DID: True
NFT is pending for a transaction: False

URIs:
   https://i.ibb.co/BtKb8sh/Block-Chian.png

Metadata URIs:

License URIs:
```

**Comment by FreeKopcap** [comment]
1.8.0-b1 still has the same bug

**Comment by FreeKopcap** [comment]
I've been advised to clear NFT cache in Settings.
Actually cache cleaning did not help but I discovered a checked option “Hide objectionable content” in Settings. When I checked it off - NFTs appeared!
Don’t know why: I do not see any “objectionable content” category in their description…
Could your show me - where is objectionable content, for example, in this NFT: nft1fq4dd0xnn7r82c4qlsuh04uy32nlkkmymvawnpwsj5k37mmlay8q304af6
